src/humancellatlas/data/metadata/api.py

```diff
@@ -744,6 +744,7 @@ class File(LinkedEntity):
     to_processes: MutableMapping[UUID4, Process]
     manifest_entry: ManifestEntry
     content_description: Set[str]
+    source: str
```

Comment:
```suggestion
    file_source: str
```

So as not to conflict with Azul's notion of `source` aka snapshot for TDR.